praw/models/reddit/subreddit.py

```diff
@@ -2174,6 +2179,84 @@ def _upload_image(self, image_path, data):
                 raise APIException(error_type, error_value, None)
             return response
 
+    def _upload_style_asset(self, image_path, data):
```

Comment:
It seems like the `data` param is always a dictionary with the key `'imagetype'`. Is there a possibility that the value of `data` could be different in the future? I ask because if things are going to stay as they are, I think it makes more sense to have an `image_type` param instead of `data` and create `data = dict()` inside the method. That also avoids the potential problem of mutating a function argument.

Reply:
Separating the upload of the asset from the modification of the structured_style data has, I think, eliminated the possibility that we might pass anything other than 'imagetype' to the upload method.  I've made the parameter explicit.